skeleton/skeletonizer.cpp

```diff
@@ -1776,7 +1777,7 @@ void Skeletonizer::toggleNodeSelection(const QSet<nodeListElement*> & nodeSet) {
 }
 
 void Skeletonizer::selectNodes(QSet<nodeListElement*> nodeSet) {
-    select(nodeSet);
+    select(std::move(nodeSet));
```

Comment:
here is the question why the argument isn’t a `const &` in the first place
i don’t know, would have to think about it

Reply:
`select` modifies the set, so it’s intentional